cmd/describe.go

```diff
@@ -62,6 +69,28 @@ var describeCmd = &cobra.Command{
 
 func init() {
 	GetRootCmd().AddCommand(describeCmd)
+	describeCmd.PersistentFlags().BoolVarP(&Json, "json", "J", false, "display JWT body as JSON")
 	describeCmd.PersistentFlags().BoolVarP(&WideFlag, "long-ids", "W", false, "display account ids on imports")
 	describeCmd.PersistentFlags().BoolVarP(&Raw, "raw", "R", false, "output the raw JWT (exclusive of long-ids)")
 }
+
+func bodyAsJson(data []byte) ([]byte, error) {
+	chunks := bytes.Split(data, []byte{'.'})
```

Comment:
Since I don't know the format of `data` here, I can't really comment on the validity of this parsing.

Reply:
this is just JWT parsing - there are three JSON objects separated by '.'.